packages/client-sdk-nodejs/src/internal/cache-data-client.ts

```diff
@@ -1447,7 +1453,7 @@ export class CacheDataClient implements IDataClient {
                     resolve(
                       new CacheGet.Error(
                         new InvalidArgumentError(
-                          'Compressor is not set, but decompress was configured; please install @gomomento/sdk-nodejs-compression and call `Configuration.withCompressionStrategy` to enable compression.'
+                          'Compressor is not set, but `CacheClient.get` was called with the `decompress` option; please install @gomomento/sdk-nodejs-compression and call `Configuration.withCompressionStrategy` to enable compression.'
```

Comment:
See almost same error thrown over and over again. Maybe worth making specific CompressionError type reduce copy pasta? Can be in follow up PR.

Reply:
Totally agreed on DRYing up the code. With the integration tests this should be straightforward.